Misc/NEWS.d/next/Library/2021-08-10-00-05-53.bpo-44859.9e9_3V.rst

```diff
@@ -0,0 +1,10 @@
+Raise more accurate and :pep:`249` compatible exceptions in :mod:`sqlite3`.
+
+* Raise :exc:`~sqlite3.InterfaceError` instead of
+  :exc:`~sqlite3.ProgrammingError` for ``SQLITE_MISUSE`` errors.
```

Comment:
You might run into https://github.com/python/core-workflow/issues/394 here

Reply:
Hm, I manually edited the NEWS file after blurb generated it, so I think I'm fine. I did a `make html` just to be sure, and the changelog was formatted correctly.

Reply:
Confirmed that it works right, I wonder what changed.